app/login.js

```diff
@@ -28,5 +29,5 @@ define([], function () {
 	    }
 	  }
 	};
-	electronsudo.exec("chmod 0777 database/user.sqlite", options, function(error) {});
+	electronsudo.exec("chmod 755 database/", options, function(error) {});
```

Comment:
Why does the owner need to be able to execute it? Shouldn't it be  500? Only the owner needs to read and write, right?